infra/src/main/kotlin/fi/fta/geoviite/infra/publication/PublicationService.kt

```diff
@@ -705,12 +704,15 @@ class PublicationService @Autowired constructor(
     @Transactional(readOnly = true)
     fun getPublicationDetailsAsTableItems(id: IntId<Publication>): List<PublicationTableItem> {
         logger.serviceCall("getPublicationDetailsAsTableRows", "id" to id)
+        val geocodingContextCache = mutableMapOf<Instant, MutableMap<IntId<TrackLayoutTrackNumber>, GeocodingContext>>()
```

Comment:
Tämä kakku meni jonkin verran uusiksi Jyrkin kommenttien perusteella. Täällä luodaan itse kakku ja eteenpäin passataan vaan täällä määritelty klosuuri